<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customeradder</class>
 <widget class="QDialog" name="customeradder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color: rgb(34, 40, 49);
	
}



QDialog
{
	background-color: rgb(238, 238, 238);
}



QLabel
{
	background-color:rgb(238, 238, 238);
	color: rgb(34, 40, 49);
	font: 11pt &quot;DejaVu Sans Mono&quot;;
}

QLineEdit
{

	color:rgb(238, 238, 238);
	background-color: rgb(34, 40, 49);
	border-radius: 9px;
	font: 9pt &quot;DejaVu Sans Mono&quot;;

}


QPushButton
{
	border-radius: 12px;
	background-color: rgb(255, 211, 105);
	color: rgb(34, 40, 49);
	font: 700 11pt &quot;DejaVu Sans Mono&quot;;
	
	
}

</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>0</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/*background-color: rgb(0, 0, 86);*/</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>18</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 238);
background-color:rgb(34, 40, 49);
font: 700 18pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Customer Add</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>240</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorDisplay">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>171</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);
color: rgb(224, 27, 36);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numberIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="addressIn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addCustomer">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>490</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearScreen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>540</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addedSucessfully">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>600</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans Mono</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(238, 238, 238) ;
color: rgb(192, 28, 40);
font: 9pt &quot;DejaVu Sans Mono&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>addressIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>252</x>
     <y>436</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>nameIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>553</y>
    </hint>
    <hint type="destinationlabel">
     <x>378</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>numberIn</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearScreen</sender>
   <signal>clicked()</signal>
   <receiver>errorDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>errorDisplay</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>374</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>numberIn</sender>
   <signal>textChanged(QString)</signal>
   <receiver>addedSucessfully</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
